onnxruntime/server/util.h

```diff
@@ -10,6 +10,34 @@
 namespace onnxruntime {
 namespace server {
 
+/**
+ * A RAII container for MLValue buffers
+ */
+class MemBufferArray {
+ public:
+  MemBufferArray() = default;
+
+  uint8_t* AllocNewBuffer(size_t tensor_length) {
+    auto* data = new uint8_t[tensor_length];
```

Comment:
if the `tensor_length` is 0, we should return a `nullptr`.

Reply:
As we mentioned offline, this is checked [here](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/core/framework/tensorprotoutils.cc#L428)